frontend/packages/policy-editor/src/components/PolicyEditorTabs/PolicyEditorTabs.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { PolicyEditorTabs } from './PolicyEditorTabs';
+import { PolicyEditorContext } from '@altinn/policy-editor/contexts/PolicyEditorContext';
```

Comment:
Should use relative import here 😄 
```suggestion
import { PolicyEditorContext } from '../../../contexts/PolicyEditorContext';
```